dll/win32/shell32/folders/CDesktopFolder.cpp

```diff
@@ -796,9 +835,10 @@ HRESULT WINAPI CDesktopFolder::GetDetailsOf(
     if (FAILED_UNEXPECTEDLY(hr))
         return hr;
 
-    hr =  psf->GetDetailsOf(pidl, iColumn, psd);
-    if (FAILED_UNEXPECTEDLY(hr))
```

Comment:
Is dropping `FAILED_UNEXPECTEDLY()` intended?

Reply:
Yes. The last three lines were as follows:
```
    if (FAILED_UNEXPECTEDLY(hr))
        return hr;

    return hr;
    ```
    So the final result is really just the same as ```return hr;``` which is simplier.

Reply:
The result is not the same in the debug build because you cut out a debug print. The original will print if GetDetailsOf fails.

Reply:
@whindsaks, you make a good point that I had not realized. I have restored the original code in this area. Thanks.